### PR TITLE
[dev-ratio] update R template for propr local modules

### PR DESCRIPTION
**Pull request by suzannejin**
nf-core/propr is outdated.
- [x] add a temporary local module for the new propr/propd code (will be updated into nf-core during hackathon).
- [x] add a temporary local module for the new propr/propr code
- [x] add a temporary local module for the new propr/grea code (container is not updated, so not enabled inside the test_experimental yet)
- [ ] the linting works for the old version, but not for tools v3.0.0, so I skip it for the moment.

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/differentialabundance/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/differentialabundance _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nf-test test main.nf.test -profile test,docker`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by bjlang** [comment]
@nf-core-bot fix linting

File: conf/test_experimental.config [hunk 1]
```diff
@@ -28,7 +28,7 @@ params {
     matrix =  'https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/genomics/mus_musculus/rnaseq_expression/SRP254919.salmon.merged.gene_counts.top1000cov.tsv'
     contrasts =  'https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/genomics/mus_musculus/rnaseq_expression/SRP254919.contrasts.csv'
     tools = "$projectDir/assets/tools_samplesheet.csv"
-    pathway = "diff_prop,diff_prop_noperm,filtered_pcor,prop"
+    pathway = "propd"
```
Comment (bjlang) [inline conf/test_experimental.config]: I think the test should be as exhaustive as possible, i.e. test as many pathways as possible

File: subworkflows/local/differential/main.nf [hunk 2]
```diff
@@ -72,7 +77,9 @@ workflow DIFFERENTIAL {
     //     .mix(DESEQ2_DIFFERENTIAL.out.results)
 
     emit:
-    results   = ch_results
-    adjacency = ch_adjacency
-    // TODO add filtered results channel
+    results_pairwise          = ch_results_pairwise
+    results_pairwise_filtered = ch_results_pairwise_filtered
+    results_genewise          = ch_results_genewise
```
Comment (bjlang) [inline subworkflows/local/differential/main.nf:83]: I believe `ch_results_genewise` should also be filled with something. IMO it's quite unintuitive that the `ch_results_genewise` would be empty while `ch_results_genewise_filtered` is not.

Comment (suzannejin) [reply subworkflows/local/differential/main.nf:83]: i have added now :)

**Comment by WackerO** [comment]
Hey, the PR looks good to me, thanks! But I'd like to remind you to please split the changes into several smaller PRs when you prepare the merge of your changes into the dev branch, that way it is much easier to review 😃

**Review by WackerO — APPROVED** [review]

**Pull request merged**